### DISCUSSION TITLE
Convert a 24/96 FLAC to 16/44 FLAC

### DISCUSSION THREAD

**Discussion by tritrek** [enzo1982/freac#633] [Audio Format Discussion]
Hi, I just can't seem to find where to set FLAC encoder settings... if you convert anything (but FLAC...) to FLAC, it spits out 16/44 FLAC but if you try to convert any FLAC, it just passes it through, doesn't convert it to anything. If it was a 24/96, it comes out as 24/96.... very weird.
Using the freac for Mac.
Thanks....

**Comment by zjDrummond** [comment]
Did you ever figure this out? I've done this sort of conversion myself for smaller CD quality files. I used the sample format, and sample rate converters under settings > processing > components to do that. Did you try that?

**Reply by tritrek** [reply]
I have tried pretty much everything but then abandoned it. To be specific, yes I have tried that setting as well. If I don't set anything, it simply encodes it to exactly what it was, that is, if it was a 24/96, it encodes it to 24/96 (no use...), so, now I have created those those settings:
Sample format converter :: Number format: integer // Sample resolution: 16bit // Dithering : Dither type: TPDF
Sample rate converter :: Converter: Best sinc interpolator // Output sample rate : Sampling rate: 44100

Guess what? LOL. It encoded it to 32/44 instead of 16/44, so basically the same size and again, no use.
I think it's either buggy or just bad coding... a shame, practically the only handy converter app for Mac...
